variables.tf

```diff
@@ -1,37 +1,40 @@
 variable "additional_endpoint_arns" {
   description = "Any alert endpoints, such as autoscaling, or app scaling endpoint arns that will respond to an alert"
   default     = []
-  type        = "list"
+  type        = list(string)
 }
 
 variable "sns_topic_arn" {
   description = "An SNS topic ARN that has already been created. Its policy must already allow access from CloudWatch Alarms, or set `add_sns_policy` to `true`"
   default     = ""
-  type        = "string"
+  type        = string
 }
 
 variable "add_sns_policy" {
   description = "Attach a policy that allows the notifications through to the SNS topic endpoint"
   default     = "false"
-  type        = "string"
+  type        = string
 }
 
-variable "region" {
-  description = "The region that should be monitored for unauthorised AWS API Access. Current region used if none provied."
+variable "log_group_region" {
```

Comment:
Any reason to change the name of this?

Reply:
`region` seemed like it was slightly inaccurate since the resources in this module don't need to pick up from the same region that the log group might be in.  For example, if you had a cloudtrail in region x but wanted to setup the alerts in a region y to monitor them, then it's more obvious to have the `region` be where this module gets deployed, and the input of `log_group_region` be where the cloudtrail region is that it is monitoring.

Let me know if that doesn't make sense though

Reply:
Yeah, I can understand the thought process. I'm a little on the fence, but I'm good with it I think.